### PR TITLE
Feature/officer change form error validation

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/integration/change_officer/change_corporate_secretary_details.spec.js [hunk 1]
```diff
@@ -67,25 +63,28 @@ describe('Change corporate secretary details - CH04', () => {
 
         // Check to ensure Tick and Cross are displayed
         const preFiling = new GenericPreFilingPage();
-        preFiling.checkPageIsDisplayedCorrectly();
+        //preFiling.checkPageIsDisplayedCorrectly();
```
Comment (rsuller) [inline cypress/integration/change_officer/change_corporate_secretary_details.spec.js:66]: Needed?

File: cypress/support/page_objects/generic/Address.js [hunk 2]
```diff
@@ -30,46 +30,44 @@ class AddressPage extends BasePage {
     }
 
     lookUpAddress(propertyNumber, postcode, propElement, postcodeElement, lookupButton) {
-        cy.get(propElement).type(propertyNumber)
-        cy.get(postcodeElement).type(postcode)
+        cy.get(propElement).clear().type(propertyNumber)
```
Comment (rsuller) [inline cypress/support/page_objects/generic/Address.js:33]: Worth wrapping up a method to clear and type into element in the `BasePage`

Comment (danfalc) [reply cypress/support/page_objects/generic/Address.js:33]: Something like this? 

`    clearThenTypeIntoElement(selector) {
        element = cy.get(selector);
        if (!element.should('be.empty')) {
            element.clear();
        }
        element.type();
        return this;
    }`

Worth doing in a separate refactoring PR as it'll affect other areas?

Comment (danfalc) [reply cypress/support/page_objects/generic/Address.js:33]: Sorry formatting is crap there

Comment (rsuller) [reply cypress/support/page_objects/generic/Address.js:33]: Yeah OK, hold off for a separate refactor as to be honest there may be other areas of potential improvement. This has now grown quite a bit over time 😄

**Pull request merged**